motor-control/tests/test_limit_switch.cpp

```diff
@@ -91,6 +95,8 @@ TEST_CASE("Move with stop condition != limit switch") {
         test_objs.hw.set_mock_lim_sw(true);
         CHECK(!test_objs.handler.pulse());
         WHEN("the move is finished") {
+            REQUIRE(test_objs.handler.get_current_position() !=
```

Comment:
yes, this check is just to ensure that the position doesn't get set to some arbitrary very high number at the beginning of any move that _isn't_ a home move though. Are you referring to the home completed in the other test case?

Reply:
I see. I didn't understand what this was trying to test upon looking at it. It might make more sense to verify that the position is equal to distance * 2 (for example) if you did two simultaneous non-homing moves back to back. Or, you could do a sequence of moves (such as non-home, home, non-home, non-home) to feel confident.